tests/tck/features/schema/Comment.feature

```diff
@@ -69,20 +69,20 @@ Feature: Schema Comment
       CREATE SPACE test_comment_empty (vid_type=FIXED_STRING(8)) comment = '';
       """
     Then the execution should be successful
-    When try to execute query:
-      """
-      SHOW CREATE SPACE test_comment_empty;
-      """
-    Then the result should be, in any order:
-      | Space                | Create Space                                                                                                                                                                                    |
-      | "test_comment_empty" | "CREATE SPACE `test_comment_empty` (partition_num = 100, replica_factor = 1, charset = utf8, collate = utf8_bin, vid_type = FIXED_STRING(8), atomic_edge = false) ON default_zone comment = ''" |
-    When executing query:
-      """
-      DESC SPACE test_comment_empty;
-      """
-    Then the result should be, in any order:
-      | ID    | Name                 | Partition Number | Replica Factor | Charset | Collate    | Vid Type          | Atomic Edge | Zones          | Comment |
-      | /\d+/ | "test_comment_empty" | 100              | 1              | "utf8"  | "utf8_bin" | "FIXED_STRING(8)" | false       | "default_zone" | ""      |
+    # When try to execute query:
+    # """
+    # SHOW CREATE SPACE test_comment_empty;
```

Comment:
Why disable this test? We should fix it.

Reply:
because the zone name is automatically generated by address we can't control

Reply:
We could use regexp

Reply:
fixed and please go on ~